app/views/layouts/_header.html.erb

```diff
@@ -6,7 +6,7 @@
             <li><%= link_to "About", about_path %></li>
             <li><%= link_to "Contact Us", contact_path %></li>
             <% if logged_in? %>
-            <li><%= link_to "Users", '#' %></li>
+                <li><%= link_to "Users", users_path %></li>
```

Comment:
@johnedelmapa 
The whole block should be updated, in this screenshot, you could see the `if/else` block is hard to read
![image](https://user-images.githubusercontent.com/20507565/61426979-f38a9680-a94e-11e9-8349-c36873dea160.png)